### DISCUSSION TITLE
Explication du paramètre "Puissance de l'équipement"

### DISCUSSION THREAD

**Discussion by msd3287** [jmcollin78/versatile_thermostat#741] [Q&A] [answered]
Bonjour,
Dans les principaux attributs du Vtherm on retrouve le paramètre "Puissance de l'équipement" (Device power) qui est à 1 par défaut. 
Est-ce que ce paramètre correspond à la puissance du radiateur en W ou en KW, ou bien le 1 (à adapter, mais quoi mettre?) correspond-t-il à un facteur multiplicateur.
Est-ce utile de renseigner ici la puissance du radiateur, ou d'adapter le coefficient si c'est de cela qu'il s'agit, dans le cas d'un chauffage central eau chaude (chaudière gaz/fioul + radiateurs) pour que le calcul du capteur "Energy" du Vtherm soit correctement adapté ?
Je n'ai pas retrouvé d'explications dans la doc.
Merci pour les éclaircissements.

**Comment by jmcollin78** [comment, marked as answer]
Hello,

Cf. https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/base-attributes.md#choix-des-attributs-de-base

Il faut mettre le total de toutes les puissances des équipements controlés par le VTherm. L'unité doit être la même que celle utilisée pour la fonction de délestage (cf. https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/feature-power.md)
